src/components/menus/base-equipment-menu.js

```diff
@@ -152,6 +152,7 @@ const BaseEquipmentMenu = ({
     const equipmentsNotDeletable = [
         equipments.lccConverterStations,
         equipments.vscConverterStations,
+        equipments.hvdcLines,
```

Comment:
Do the PO explictly want to wait before making the "hvdc delete from the map" available ?

Reply:
Discussed: we keep it hidden for the moment